samples/create_world_cities.py

```diff
@@ -0,0 +1,43 @@
+import sys
+sys.path.append('..')
+
+import api
+import util
+import acl
+import json
+
+
+def create_world_cities(args):
+    kwargs = vars(args)
+    print "Creating dataset"
+    dataset_meta = api.create_dataset('world-cities-data', **kwargs)
+    print "Ingesting data from CSV"
+    util.ingest_file(dataset_meta['id'], csv='simplemaps-worldcities-basic.csv', kind='city', answer_yes=True, generate_ids=True, **kwargs)
```

Comment:
A better sample would not use util, but there are other commands used here that are not yet covered by documentation.  At some point this should be replaced with a local function that builds the entities.